### DISCUSSION TITLE
HM1500, Direktanschluss 2 Panels+2 Panels an Akku

### DISCUSSION THREAD

**Discussion by marco198511** [hoylabs/OpenDTU-OnBattery#517] [Q&A]
Hallo,

ich habe aktuell 4x 410W Panels(2xOst + 2x West) direkt an einem HM1500(gedrosselt auf 600w) am Netz.

Zusätzlich habe ich zum Test 2x 350w Panels an einem Victron 100/20 (läd einen 24V 100AH Liefpo4) und einem SUN1000 WR Mit Trucky2Shelly Board zur Nulleinspeisung. Dies Funktioniert Top.
Zum Auslesen der benötigtem Leistung ist ein Shelly3EM im Einsatz.

Der Sun1000 soll allerdings wieder weichen da ich doch Gesetzeskonform (Deutschland) unterwegs sein, und nur 600w bzw. später wenn erlaubt 800w Leistung erreichen möchte.

Also soll die Batterie inkl. Victron an den HM1500 wechseln.

ich habe bereits eine 2. gleiche Batterie im Zulauf damit ich auf ein 48V System umsteigen kann (außerdem waren die 2,5kwh etwas zu wenig)
meine Frage wäre, der Victron 100/20 schafft ja nicht die 4 panels aufzunehmen, wäre es möglich weiterhin 2 Panels direkt am WR zu betreiben und die anderen 2 Eingänge mit dem 48v akkusystem zu verbinden? wie reagiert hier die Steuerung des opendtu on Battery?

Ich würde den Hoymiles am AC Ausgang schalten falls der Akku leer ist, damit die Kondensatoren nicht in Mitleidenschaft gezogen werden bei Abrupter Spannungsversorgung, beim wieder zuschalten der Batterie!

Wenn dies nicht möglich ist müsste ich auf einen größeren Victron Regler wechseln, bzw. noch einen 2. 100/20 für die restlichen 2 Panels. Dann wäre allerdings auch die frage wie die openDTU mit mehreren Victrons klarkommt.

ich hoffe mein problem ist verständlich ;-)

Gruß Marco

**Comment by schlimmchen** [comment]
> wäre es möglich weiterhin 2 Panels direkt am WR zu betreiben und die anderen 2 Eingänge mit dem 48v akkusystem zu verbinden?

Nein, das wird nicht unterstützt. Der HM-1500 wird in diesem Setup nur dann "zufällig" deinen Haushaltsverbrauch genau decken, wenn die Solarpanele genügend Leistung liefern können. Nachts geht das dann in die Hose, weil nur ungefähr 50% des Leistungslimits erreicht werden können. Das Limit wird dann vermutlich skaliert und es kommt vielleicht doch irgendwie hin, aber ganz ehrlich, ich möchte dir davon dringend abraten. Auch elektrisch. Das klappt ja sowieso nur, wenn du die zwei Panele am einen MPPT anschließt, und die Batterie am anderen (Eingang 1 und 2 sowie 3 und 4 teilen sich jeweils einen MPPT). Das ist ein Gefummel. Lass es bitte.

> Dann wäre allerdings auch die frage wie die openDTU mit mehreren Victrons klarkommt.

Das wird zur Zeit leider auch nicht unterstützt, aber das wird kommen, und wir sind nicht mehr weit davon entfernt. Jedenfalls bis zur nächsten Saison stehen die Chancen sehr gut, dass das unterstützt wird.

**Reply by Manos1966** [reply]
> ber das wird kommen, und wir sind nicht mehr weit davon entfernt. Jedenfalls bis zur nächsten Saison stehen die Chancen sehr gut, dass das unterstützt wird.

soll anfangen zu beten, das bald ESP32 mit 50pins gibt? :-D


> Der HM-1500 wird in diesem Setup nur dann "zufällig" deinen Haushaltsverbrauch genau decken, wenn die Solarpanele genügend Leistung liefern können. Nachts geht das dann in die Hose, weil nur ungefähr 50% des Leistungslimits erreicht werden können. Das Limit wird dann vermutlich skaliert und es kommt vielleicht doch irgendwie hin, aber ganz ehrlich, ich möchte dir davon dringend abraten. Auch elektrisch. Das klappt ja sowieso nur, wenn du die zwei Panele am einen MPPT anschließt, und die Batterie am anderen (Eingang 1 und 2 sowie 3 und 4 teilen sich jeweils einen MPPT).

Guter Punkt, muss ich irgendwo in die Wiki addieren....hier:
|https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/FAQ
@schlimmchen kannst du bei Gelegenheit ein Blink drauf werfen, ich hoffe ich konnte mich korrekt erinnern wie die HOYMILES das Limit teilt. Es gab aber auch ein Unterschied zwischen Prozent vs Absolute Zahl glaube ich.... 
Erinnert sich jemand die Diskussion # wo es ausfuehrlich darueber diskutiert wurde (wie das Limit "interpretiert" wird)?

**Reply by schlimmchen** [reply]
> soll anfangen zu beten, das bald ESP32 mit 50pins gibt? :-D

Du braucht einen Pin pro MPPT... **Einen**... Vielleicht zwei falls wir in Zukunft auch Anfragen schicken.

> kannst du bei Gelegenheit ein Blink drauf werfen

Ne, weil ich davon keine Ahnung habe. Für mich ist das nicht nachvollziehbar, was der Wechselrichter dann macht. Ich hab's mal experimentell bestimmen wollen und hab dann beschlossen, dass das Zeitverschwendung ist. Ich begreif's nicht.

**Comment by Manos1966** [comment]
@marco198511 

> der Victron 100/20 schafft ja nicht die 4 panels aufzunehmen, 

Oh doch!
Ich finde meine Notizen nicht gerade, aber eine 2P2S Konfiguration wird mit dem MPTT 100/20 funktionieren.

Nachteile: 
- doppelte Amperes = hoehere Verluste abhaengig KabelLaenge und Querschnitt (verglichen zu der Moeglichkeit alle vier Paneele 4S = doppelte Spannung (V) )
Der Victron muss ganz nah an der Batterie sein weil niedrig Volt (52V) und hohe Amperes.
- Der 100/20 wird max 20Amperes liefern (Amperes max deine Paneele wird etwa 12A sein, also 2x12=24A, also 4A gehen verloren) 

Fazit: Du hast "Pandoras Buchse" aufgemacht.... Viel Spass beim Lesen: Verluste Kabel Querschnitt / Amperes / Volts / Kabellänge  

Aber 2P2S beim Victron 100/20 wird gehen ;-)

**Reply by schlimmchen** [reply]
> Ich finde meine Notizen nicht gerade, aber eine 2P2S Konfiguration wird mit dem MPTT 100/20 funktionieren.

Das kannst du ohne das Datenblatt der Solarmodule zu kennen unmöglich zusichern. Das hängt ja von deren Eigenschaften ab. Also mit meinen Modulen gingen zwei in Serie und zwei parallel nicht, weil der Kurzschlussstrom **deutlich** zu hoch wäre.

> Der 100/20 wird max 20Amperes liefern (Amperes max deine Paneele wird etwa 12A sein, also 2x12=24A, also 4A gehen verloren)

Das ist nicht zutreffend. Die 20A beziehen sich auf den Batterieladestrom. Die Panele haben damit erstmal nichts zu tun. Das Ding kann laut seinem Datenblatt höchstens 1160W PV-Leistung verarbeiten. Bei 74V sind das ca. 15A.

**Reply by Manos1966** [reply]
Danke @schlimmchen 
kann ich davon ausgehen, du hast "ältere PV Module"? (bei den neuen sehe ich selten Spannung über 40V)

Ich habe das Victron Datenblatt gefunden 👍 Sehr Interessant, ich habe nie das klein gedruckte gelesen!
...und brauche etwas Hilfe 🙈

- Max PV-Nennleistung bei 48V Batterie 1160W (hoeher als das, wird automatisch von 100/20 begrenzt)
Vier Paneele mit 410Wp sind 1640Wp, was in der Praxis bedeutet 70% bis 80% im Winter? also 1148W - 1312W 
Dann wurde ich behaupten, mit 1160W PV Eingang, der Victron 100/20 ist zwar ausgereizt, aber doch noch geeignet.

- Max Batteriestrom ist 20A. Da eine 48V Pylontech Batterie mit 52V geladen werden sollte  (laut Victron)  20A x 52V = 1040W mit dieser Max Batterie-Ladeleistung kann man zufrieden sein im Winter (wobei, falls die Solar Module 80% erreichen, dies bedeutet ca 25% Verlust (1312W PV-Leistung versus 1040W Batterie-Ladeleistung bei 98% Wirkungsgrad) ).

- Max PV-Kurzschlussstrom 20A Das ist mir neu! 
"Ein hoeherer Kurzschlussstrom kann bei **verpoltem Anschluss** der PV-Anlage den Victron beschaedigen"
OK, also vorausgesetzt man macht keine Fehler (also kein verpolen), ist es kein Problem wenn die PV-Anlage mehr Strom A produziert, richtig? (so habe ich es immer verstanden).
weil die englische Version sagt "A PV array with a higher short circuit current may damage the controller", ohne Verpolung zu erwaehnen.

Grundsaetzlich finde ich den 150/35 besser, weil man (ich) kann drei Panels (oder mehr) in Reihe anschliessen (circa 3x40V=120V), waehrend ein 100/20 kann nicht mehr als zwei (2x40V=80V) und Kabel-Verluste sind bedeutsam hoeher je kleiner die Spannung und je hoeher der Strom.

Das teste ich gerade, was ist wirtschaftlich besser (wohl wissend das ein 150/35 kostet genauso viel wie zwei 100/20):
- Victron 150/35 mit drei PV Panels 3S oder
- Zwei Victron 100/20 mit jeweils vier PV-Panels 2S2P (8 PV Panels insgesamt)

**UPDATE:** Falls die Feststellung #513 fuer alle 100/20 zutrifft, ist der Victron 100/20 sowieso nicht mehr geeignet.

![Victron 100-20 bis 150-35 Max Limits PV input](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/af48e9b5-d988-4fb0-a62e-00d9113c66bc)

**Reply by schlimmchen** [reply]
> kann ich davon ausgehen, du hast "ältere PV Module"? (bei den neuen sehe ich selten Spannung über 40V)

Nein, kannst du nicht. Meine Panele sind von diesem Jahr. Es sind bifaziale Luxor M120 380Wp Module. Die haben eine Leerlaufspannung von 44,5V laut Datenblatt.

Hast du außerdem den negativen Temperaturkoeffizienten im Blick? Bei -20°C haben diese Module zimelich genau 50V Leerlaufspannung.

**Reply by Manos1966** [reply]
@schlimmchen Hmm... ich werde trotzdem nicht schlau.
Wenn ich also richtig verstanden habe, sogar bei -20°C und 1000W/m2 Einstrahlung geben deine Module 50V max.
Dadurch waeren doch 2S2P moeglich (der 100/20 wurde die maximale Leerlaufspannung von 100V aushalten).
Oder habe ich irgendwo einen Denkfehler?

**Reply by schlimmchen** [reply]
Ich hab von der Spannung erzählt, weil du behauptet hast 40V Leerlaufspannung könne nur zu einem alten Modul passen.

Der MPPT 100|20 wäre für mein Setup deshalb nicht geeignet, weil die Module einen Kurzschlusstrom von 10,71A bei STC haben. Bei einem rückseitigen Ertrag von 15% sind es schon 12,32A. Da ich weiß dass meine Module einen rückseitigen Ertrag von über 30% schaffen, sind es ca. 14A. Zwei Strings parallel ergibt dann 28A. Das ist beinahe 50% über den 20, die der MPPT 100|20 verträgt.

Und ursprünglich ging es darum, dass ich klarstellen wollte, dass du nicht pauschal sagen kannst, dass der MPPT 100|20 eine Konfiguration mit zwei Modulen in Reihe und die parallel verträgt. Das hängt von den Modulen ab.

**Reply by Manos1966** [reply]
Alles klar jetzt, Danke!
(schoene Paneele!!! mit rückseitigen Ertrag von über 30%)

**Reply by marco198511** [reply]
meine Panele sind Jingli 410 w Module Glas/Glas Bifazial.

Spannung bei Max MTTP 13,29A, das wären dann 26,58A bei Parallelschaltung das möchte ich dem kleinen Victron doch nicht zumuten.

Ich würde auch sagen das der Satz "Ein hoeherer Kurzschlussstrom kann bei verpoltem Anschluss der PV-Anlage den Victron beschaedigen" ein Übersetzungsfehler ist. 
Auf 1 Victron für alle 4 Module zu setzen ist auch keine Gute Idee denn wie oben erwähnt ist dies eine OST WEST Anlage da wird der EINE MTTP warscheinlich nicht gut klarkommen.

**Comment by marco198511** [comment]
> > wäre es möglich weiterhin 2 Panels direkt am WR zu betreiben und die anderen 2 Eingänge mit dem 48v akkusystem zu verbinden?
> 
> Nein, das wird nicht unterstützt. Der HM-1500 wird in diesem Setup nur dann "zufällig" deinen Haushaltsverbrauch genau decken, wenn die Solarpanele genügend Leistung liefern können. Nachts geht das dann in die Hose, weil nur ungefähr 50% des Leistungslimits erreicht werden können. Das Limit wird dann vermutlich skaliert und es kommt vielleicht doch irgendwie hin, aber ganz ehrlich, ich möchte dir davon dringend abraten. Auch elektrisch. Das klappt ja sowieso nur, wenn du die zwei Panele am einen MPPT anschließt, und die Batterie am anderen (Eingang 1 und 2 sowie 3 und 4 teilen sich jeweils einen MPPT). Das ist ein Gefummel. Lass es bitte.
> 
> > Dann wäre allerdings auch die frage wie die openDTU mit mehreren Victrons klarkommt.
> 
> Das wird zur Zeit leider auch nicht unterstützt, aber das wird kommen, und wir sind nicht mehr weit davon entfernt. Jedenfalls bis zur nächsten Saison stehen die Chancen sehr gut, dass das unterstützt wird.


Das habe ich mir schon Gedacht das der dann auf 50% läuft und ziemlich lange nachregelt damit es irgendwie passt, da ja 2 Eingänge fehlen. Falls es das überhaupt so nachregelt, Optimal wäre das auf keinen Fall da das die Programmierung nicht vorsieht.

@Manos1966 Das Limit beim Hoymiles wird je MTTP geteilt, also je 50 Prozent des Limits.

Mit den 4 Panels am Victron weiß ich nicht so recht, wenn ich sehe wie warm der jetzt schon wird mit den 2 weniger Leistungsstarken  Panels bei jetzt ca. 75V und 10A.
Das Problem am den Größeren Victrons ist ja ebenfalls das die keinen Lastausgang haben und den Virtuellen Lastausgang kann ich nicht nutzen da dort die VE.Direkt Signale für die DTU drüber laufen.

Der Victron ist nicht weit von der Batterie entfernt, aktuell vielleicht 50cm(kabel sind 10mm² Feinadrig), das wird dann im fertigen Zustand noch ein paar cm weniger denke ich.

 Vielleicht funtkioniert das auch mit 2 Victrons, die kann/muss man ja auch koppeln wenn die eine Batterie speisen, die DTU benötigt ja grundlegend nur die Spannung des Akkus damit sie weiß ob da Leistung drauß gezogen werden kann oder nicht. Der passthrouh wenn der Akku voll ist und die Module noch Strom liefern, könnte auch funktionieren da die Victrons dann ja im Verbund auch die Leistung aufrechnen, selbst wenn nicht Regeln die runter und die DTU nimmt nur den einen als Berechnungsgrundlage das wäre mir auch egal, die ganze Geschichte ist ja eh bei 600 oder 800w abgeriegelt bei 1,6kw Panelleistung, also geht da sowieso einiges verloren wenn die Akkus voll sind. Zur Not gibt dann noch mehr Akkukapazität ;-)


Schon einmal vielen Dank für die Antworten! ich werde berichten wenn ich das angeschlossen habe und Erfahrung mit der Konfiguration gesammelt habe.

**Reply by Manos1966** [reply]
@marco198511 
Ich hoffe dein 100/20 ist Smart... hier ist ein Link wie das geht
[Laderegler Parallel anschließen VE.Smart Networking Victron](https://www.youtube.com/watch?v=rkDQcIBGIUA)

**_Noch_** funktioniert das Solar Passthrough NICHT mit zwei Victrons da die Victrons dann im Verbund auch die Leistung aufrechnen, ABER NUR EINER seiner Leistung der OpenDTU mitteilt.
...aber ich weiss, @schlimmchen hat diese Erweiterung auf seiner Liste...🙊

Die heutige Situation (nur ein Victron) hat folgendes Verhalten laut meiner Erfahrung:
- Solar Passthrough nur auf den ersten Victron = damit wird das max Ladelimit der Batterie z.B. max 90%, nicht durch die OpenDTU kontrolliert (die Batterie wird weiter geladen, so lange der zweite Victron produziert).
- Erreicht die Batterie die volle Kapazitaet, hoeren beide Victrons auf zu produzieren. Daraufhin sieht Solar Passthrough dass der erste Victron null Leistung hat und schaltet die HOYMILES komplett aus.
Wird Strom gebraucht, dann wird es von der Batterie entnommen bis die Victrons anfangen die Batterie wieder zu laden.

Ob das jetzt ein "Show-Stopper" ist, finde ich nicht. Wie ich immer sage, ich koennte 30.000EUR hin blättern und eine Anlage bekommen die alles macht... 
Dann lieber 300EUR fuer einen neuen 150/35 und meinen alten 100/20 bei Seite parken, bis die OpenDTU zwei Victrons steuern kann.
(my 2 cents)

**Reply by marco198511** [reply]
ja mein Victron ist der Smartsolar.

das Verhalten wie du es bescheibst kann ich nicht ganz nachvollziehen, aber wie gesagt ich hab das noch nicht im Betrieb.
Schaltet der die Hoymiles wirklich aus? oder regelt der nur die Leistung auf den momentanen Hausverbrauch runter. Der Solarpassthrouh ist ja auch nur ein Zusatz, wenn der Funtioniert ist das natürlich Super, wenn nicht auch ok, solange mein Hausverbrauch gedeckt ist. Normalerweise schalten die Victrons ja auch nicht direkt ab sondern regeln das runter wenn ein Verbraucher dranhängt der etwas Leistung abnimmt, so zumindest meine Erfahrung mit dem SUN1000.

**Reply by Manos1966** [reply]
Alles was du erzaehlt hast ist richtig.
Daraufhin ist es auch unwichtig ob die OpenDTU mehrere Victrons beobachtet, oder nur einen (meiner Meinung nach).

Jetzt wegen SolarPassthrough... 
SolarPassthrough (Full) regelt die OpenDTU die Hoymiles so, dass der ganze Ertrag vom VICTRON zu der HOYMILES geht. So bleibt nichts uebrig um die Batterie weiter zu laden (da die Batterie bereits voll geladen ist bzw auf das vorgegebene % Prozent Ladelimit z.B. 90%).  

Der Victron laedt die Batterie normalerweise bis 100% erreicht wurde. Danach regelt er ab (der Victron) und produziert nicht mehr (siehe blaue Pfeile im Bild).
![Verhalten 150-35](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/6e52a5d0-c942-47fd-a1e5-ac59a536eaac)

Demensprechend wird die OpenDTU den Hoymiles stoppen (**wenn kein Verbrauch da ist**).

**Reply by marco198511** [reply]
@Manos1966 ja der Victron würde abschalten wenn Kein Verbrauch da ist, das ist aber zu 99,999% nicht der Fall.

Bei mir Hängt dahinter ein ganzes Haus mit inkl Pool und Teich, da ist im Sommer mindestens 300W Grundlast am Tag.

In deinem Diagramm sieht man dieses Verhalten vom Victron auch! Der schaltet nur ein einziges mal am Tag komplett ab! Den Rest regelt er nur runter.
mal davon abgesehen wenn der die Gesamte PV Leistung versucht durchzuleiten geht das eh nicht da der Hoymiles auf 600w begrenzt bleibt.
Falls der Passthrouh doch nicht funktioniert kann man den ja auch abschalten. Wie gesagt das wäre nur die Kirsche auf der Torte.
Solange mein Hausverbrauch abgedeckt ist(bis 600w) ist dies Schon gut, zumal wenn ich sehe was diesen Sommer hier schon für Spannungen im Netz waren und der Hoymiles sogar schon wegen Überspannung abgeschalten hat, werde ich den Passthrouh vielleicht garnicht aktivieren zumindest Zeitweise.

Mittlerweile trudeln die ganzen Restlichen Komponenten ein, ich hoffe ich schaffe das am WE aufzubauen, dann kann ich berichten wie und ob das Funktioniert ;-)

**Reply by Manos1966** [reply]
_falsch gepostet...._

**Reply by marco198511** [reply]
Welches Fehlverhalten meines 100/20 meinst du?

Wenn ich zb. 300w verbrauche ist und die Batterie Voll ist schleust der Victron ca 300w durch, solange Genug Sonne da ist um das Abzudecken. Das empfinde ich nicht als Fehlverhalten, denn der merkt ja das Leistung abgefordert wird durch die Sinkende Batteriespannung und liefert dementsprechend nach, es kann natürlich sein das der mal ein paar Augenblicke wirklich abschaltet aber solange Konstant Leistung gebraucht wird leitet der das auch durch.

**Reply by Manos1966** [reply]
Entschuldigung @marco198511 
ich habe am falschen Thread gepostet... 🤦🏻‍♂️
